src/main/java/com/conveyal/analysis/controllers/GtfsVectorTileMaker.java

```diff
@@ -0,0 +1,293 @@
+package com.conveyal.analysis.controllers;
```

Comment:
Should the GtfsVectorTileMaker be moved into the com.conveyal.gtfs package since it's not a controller?